ios/TestApp/benchmark/setup.rb

```diff
@@ -0,0 +1,60 @@
+
+require 'xcodeproj'
+require 'fileutils'
+
+puts "Current directory: #{Dir.pwd}"
+
+install_path = File.expand_path("../../../build_ios/install")
+if not Dir.exist? (install_path) 
+    raise "path doesn't exist:#{install_path}!"
+end
+xcodeproj_path = File.expand_path("../TestApp.xcodeproj")
+if not File.exist? (xcodeproj_path) 
+    raise "path doesn't exist:#{xcodeproj_path}!"
+end
+
+puts "Setting up TestApp.xcodeproj..."
+project = Xcodeproj::Project.open(xcodeproj_path)
+target = project.targets.first #TestApp
+header_search_path      = ['$(inherited)', "#{install_path}/include"]
+libraries_search_path   = ['$(inherited)', "#{install_path}/lib"]
+other_linker_flags      = ['$(inherited)', "-all_load"]
+
+target.build_configurations.each do |config|
+    config.build_settings['HEADER_SEARCH_PATHS']    = header_search_path
+    config.build_settings['LIBRARY_SEARCH_PATHS']   = libraries_search_path
+    config.build_settings['OTHER_LDFLAGS']          = other_linker_flags
+    config.build_settings['ENABLE_BITCODE']         = 'No'
+end
+
+puts "Installing the testing model..."
+model_path_src = File.expand_path("./model.pt")
+model_path_dst = File.expand_path("../model.pt")
+if not File.exist?(model_path_src)
+   raise "no model can be found!"
+end
+FileUtils.cp(model_path_src, model_path_dst)
```

Comment:
why do we need copy model.pt around? seems a bit confusing to have the same file at both "./" and "../".
can we directly use "TestApp/benchmark/model.pt" from the project? or make "trace_model.py" to save to "TestApp/model.pt" directly?

Reply:
Let's keep the model in `TestApp/benchmark`